src/executables/master.c

```diff
@@ -429,6 +432,27 @@ css_accept_old_request (CSS_CONN_ENTRY * conn, unsigned short rid, SOCKET_QUEUE_
     }
 }
 
+static int
+receive_server_info (CSS_CONN_ENTRY * conn, unsigned short rid, std::string & dbname, SERVER_TYPE & type)
+{
+  int name_length;
+  char *buffer = NULL;
+
+  if (css_receive_data (conn, rid, &buffer, &name_length, -1) == NO_ERRORS)
+    {
+      // *INDENT-OFF*
+      type = static_cast<SERVER_TYPE> ((int) buffer[0] - '0');
+      // *INDENT-ON*
+      dbname = buffer + 1;
+
+      _er_log_debug (ARG_FILE_LINE, "A server with database:'%s' of type:'%s' wants to connect to cub_master.",
+		     dbname.c_str (), type == SERVER_TYPE_PAGE ? "page" : "transaction");
```

Comment:
Log only if test mode is activated.
```suggestion
      er_log_debug (ARG_FILE_LINE, "A server with database:'%s' of type:'%s' wants to connect to cub_master.",
		    dbname.c_str (), type == SERVER_TYPE_PAGE ? "page" : "transaction");
```